frontend/src/store/queries/index.ts

```diff
@@ -25,8 +28,49 @@ const baseQuery = fetchBaseQuery({
   },
 }) as BaseQueryFn<string | FetchArgs, unknown, ErrorType>;
 
+const baseQueryWithReauth: BaseQueryFn<
+  string | FetchArgs,
+  unknown,
+  ErrorType
+> = async (args, api, extraOptions) => {
+  await mutex.waitForUnlock();
+  let result = await baseQuery(args, api, extraOptions);
+
+  if (result.error && result.error.status === 401) {
+    if (!mutex.isLocked()) {
+      const release = await mutex.acquire();
+      try {
+        const refreshResult = await baseQuery(
+          {
+            url: `${ApiPath.AUTH}/refreshToken`,
```

Comment:
`/refreshToken` also should be part of the `ApiPath`

Reply:
oh, ok